cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,111 @@
+
+    # Add CORS headers to the response
+    headers = {
+        'Access-Control-Allow-Origin': origin,
+        'Access-Control-Allow-Methods': 'POST, OPTIONS',
+        'Access-Control-Allow-Headers': 'Authorization, Content-Type',
+    }
+
+    return headers
+
+@functions_framework.http
+def token_refresh(request):
```

Ruff (0.8.2)

`token_refresh` is too complex (12 > 10)

(C901)